knowledge-base/architectural-decisions-records/circuit-breaker-options.md

```diff
@@ -0,0 +1,32 @@
+# Circuit Breaker Options
```

Comment:
The directory for this should be `architectural-decision-records` (or maybe even rename it to `adr`)

Reply:
renamed